cli/operator/keys/create.go

```diff
@@ -44,7 +44,7 @@ func CreateCmd(p prompter.Prompter) *cobra.Command {
 		Long: `
 Used to create ecdsa and bls key in local keystore
 
-keyname is required
+keyname (required) - This will be name of the created key file. It will be saved as <keyname>.ecdsa.key.json or <keyname>.bls.key.json
```

Comment:
```suggestion
keyname (required) - This will be the name of the created key file. It will be saved as <keyname>.ecdsa.key.json or <keyname>.bls.key.json
```

Sorry for the nick-picking, I hate it but for documentation it is needed.